src/rpc/misc.cpp

```diff
@@ -31,94 +31,47 @@
             "\nResult:\n"
             "{\n"
             "  \"isvalid\" : true|false,       (boolean) If the address is valid or not. If not, this is the only property returned.\n"
-            "  \"address\" : \"address\", (string) The bitcoin address validated\n"
+            "  \"address\" : \"address\",        (string) The bitcoin address validated\n"
             "  \"scriptPubKey\" : \"hex\",       (string) The hex encoded scriptPubKey generated by the address\n"
-            "  \"ismine\" : true|false,        (boolean) If the address is yours or not\n"
-            "  \"iswatchonly\" : true|false,   (boolean) If the address is watchonly\n"
             "  \"isscript\" : true|false,      (boolean) If the key is a script\n"
-            "  \"script\" : \"type\"             (string, optional) The output script type. Possible types: nonstandard, pubkey, pubkeyhash, scripthash, multisig, nulldata, witness_v0_keyhash, witness_v0_scripthash\n"
-            "  \"hex\" : \"hex\",                (string, optional) The redeemscript for the p2sh address\n"
-            "  \"addresses\"                   (string, optional) Array of addresses associated with the known redeemscript\n"
+            "  \"ismine\" : true|false,        (boolean) DEPRECATED. If the address is yours or not\n"
+            "  \"iswatchonly\" : true|false,   (boolean) DEPRECATED. If the address is watchonly\n"
+            "  \"isscript\" : true|false,      (boolean) DEPRECATED. If the key is a script\n"
+            "  \"script\" : \"type\"             (string, optional) DEPRECATED. The output script type. Possible types: nonstandard, pubkey, pubkeyhash, scripthash, multisig, nulldata, witness_v0_keyhash, witness_v0_scripthash\n"
+            "  \"hex\" : \"hex\",                (string, optional) DEPRECATED. The redeemscript for the p2sh address\n"
+            "  \"addresses\"                   (string, optional) DEPRECATED. Array of addresses associated with the known redeemscript\n"
             "    [\n"
             "      \"address\"\n"
             "      ,...\n"
             "    ]\n"
-            "  \"sigsrequired\" : xxxxx        (numeric, optional) Number of signatures required to spend multisig output\n"
-            "  \"pubkey\" : \"publickeyhex\",    (string) The hex value of the raw public key\n"
-            "  \"iscompressed\" : true|false,  (boolean) If the address is compressed\n"
+            "  \"sigsrequired\" : xxxxx        (numeric, optional) DEPRECATED. Number of signatures required to spend multisig output\n"
+            "  \"pubkey\" : \"publickeyhex\",    (string) DEPRECATED. The hex value of the raw public key\n"
+            "  \"iscompressed\" : true|false,  (boolean) DEPRECATED. If the address is compressed\n"
             "  \"account\" : \"account\"         (string) DEPRECATED. The account associated with the address, \"\" is the default account\n"
```

Comment:
You removed this from the output, but not the docs. For now I think we should leave the account display here, primarily just because we will want to keep it identical but just rename it label when we do accounts->label conversion.

Reply:
Added the accounts field back to the output.